src/renderer/uia/UiaRenderer.cpp

```diff
@@ -228,7 +261,7 @@ UiaEngine::~UiaEngine()
 // - S_FALSE since we do nothing
 [[nodiscard]] HRESULT UiaEngine::PaintBackground() noexcept
 {
-    return S_FALSE;
+    return S_OK;

```

Comment:
Why `S_OK` here and not `S_FALSE` like the other spots? (and the other 4 below this)